cfgov/legacy/housing_counselor/geocoder.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import, unicode_literals
+
+from six import print_
+
+
+def geocode_counselors(counselors, **kwargs):
+    """Fill in missing latitude/longitude data for housing counselors."""
+    return ZipCodeBasedGeocoder(**kwargs).geocode(counselors)
+
+
+class ZipCodeBasedGeocoder(object):
+    """Fill in missing latitude/longitude data using zipcode locations.
```

Comment:
Ah, darn it. You are right! There are indeed 23 hits for `"agc_ADDR_LATITUDE":"0"`. I was erroneously assuming that the lat/lng would be missing but that's apparently not the case. My bad!

To maintain compatibility with current django-hud behavior, what if I modify this so that it does the same zipcode-based geolocation as the existing functionality? The code is already written so it should just be a one line change plus some additional unit tests.